src/style.css

```
@@ -1,3 +1,72 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+  background-color: #fff;
+  font-weight: 500;
+  font-size: 12px;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+}
+
+.nav {
+  display: flex;
+  align-items: center;
+}
+
+.list {
+  list-style: none;
+  display: flex;
+  align-items: center;
+}
+
+._link {
+  display: flex;
+  height: 60px;
+  align-items: center;
+  text-decoration: none;
+  margin: 0 10px;
+  color: black;
+  text-transform: uppercase;
+}
+
+._link:hover {
+  color: #00acdc;
+}
+
+.list-item:first-child ._link {
+  margin-left: 0;
+}
+
+.list-item:last-child ._link {
+  margin-right: 0;
+}
+
+.is-active {
+  color: #00acdc;
+  position: relative;
+}
+
+.is-active::after {
+  content: ''; /* Dodanie pseudo-elementu */
+  position: absolute; /* Ustawienie absolutnej pozycji wewnątrz .is-active */
+  bottom: -4px; /* Przesunięcie o 4 piksele w dół względem rodzica (.is-active) */
+  left: 0; /* Ustawienie na początku rodzica (.is-active) */
+  width: 100%; /* Szerokość 100% rodzica (.is-active) */
+  border-bottom: 4px solid #00acdc; /* Linia podkreślenia */
+  border-radius: 2px;
+  transform: translateY(-4px);
```

you can simplify changing bottom to 0 and removing transform: 
```suggestion
  bottom: 0; /* Przesunięcie o 4 piksele w dół względem rodzica (.is-active) */
  left: 0; /* Ustawienie na początku rodzica (.is-active) */
  width: 100%; /* Szerokość 100% rodzica (.is-active) */
  border-bottom: 4px solid #00acdc; /* Linia podkreślenia */
  border-radius: 2px;
```